docs/platform/howto/unsafe-passwords.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Change unsafe passwords
+---
+
+The Aiven Platform checks your email and password combination against a database of exposed credentials every time you log in and change your password.
```

Comment:
Should this say 'every time you log in or change your password' instead of 'and'? Just checking if both actions trigger the credential check or only one of them.

Reply:
It's when you do one or the other thing, using 'and' would sound like you log in first then change your password.